python/pyarrow/serialization.pxi

```diff
@@ -0,0 +1,233 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from libcpp cimport bool as c_bool, nullptr
+from libcpp.vector cimport vector as c_vector
+from cpython.ref cimport PyObject
+from cython.operator cimport dereference as deref
+
+try:
+    import cloudpickle as pickle
+except ImportError:
+    try:
+        import cPickle as pickle
+    except ImportError:
+        import pickle
+
+from pyarrow.lib cimport Buffer, NativeFile, check_status, _RecordBatchFileWriter
```

Comment:
I believe this is unneeded because `serialization.pxi` is part of the same Cython module